src/components/Tooltip/Tooltip.tsx

```diff
@@ -289,6 +301,11 @@ const Tooltip = ({
     openPortal();
   };
 
+  const deplayedOpenPortal = useCallback(() => {
+    const timeout = setTimeout(openPortal, delay);
```

Comment:
This is currently causing a console error when the tooltip is opened. This is because the mouse event is not passed to `openPortal`.

There are two possible fixes. One is to pass the event through to the openPortal call. The second would be to upgrade to the latest version of `react-useportal` and then change the `usePortal` call above to `usePortal({ programmaticallyOpen: true})`.

Reply:
Went with the first option and passed `MouseEvent` to `openPortal `